go/beego/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  appsignal:
+    image: appsignal/agent:latest
+    ports:
+      - "8099:8099"
```

Comment:
Is it useful to have the agent's port exposed to the host? If it isn't, then the `ports` section here can be removed, and the apps can point their OpenTelemetry exporter to `agent:8099` instead of `0.0.0.0:8099`.

[same thing applies for the other projects, and for the other things that expose ports such as MySQL and Redis and Mongo]